src/patterns/help-users-contact-a-department/default/index.njk

```diff
@@ -0,0 +1,12 @@
+---
+title: Default
+layout: layout-example.njk
+---
+
+<p class="govuk-body govuk-!-font-weight-bold">Get help with your application</p>
+
+<ul class="govuk-list">
+  <li>Telephone: 020 7946 0101</li>
+  <li><span class="govuk-visually-hidden">Opening times: </span>Monday to Friday, 9am to 5pm (closed on bank holidays)</li>
+  <li class="govuk-!-margin-top-5"><a class="govuk-link" href="#">Find out about call charges</a></li>
```

Comment:
I think here where you're trying to create a distinction between the last item in the list and the other items you could consider moving it out of the list entirely, something like:

```html
<ul class="govuk-list">
  <li>Telephone: 020 7946 0101</li>
  <li><span class="govuk-visually-hidden">Opening times: </span>Monday to Friday, 9am to 5pm (closed on bank holidays)</li>
</ul>
<p class="govuk-body">
   <a class="govuk-link" href="#">Find out about call charges</a>
</li>
```

Reply:
In the overall guidance you recommend linking to `https://www.gov.uk/call-charges`, could we consider using that for the links in the example instead of `#`?

Reply:
For security reasons, we can linked to GOV.UK in frame.